pkg/service/credential/storage.go

```diff
@@ -21,17 +20,15 @@ import (
 )
 
 type StoreCredentialRequest struct {
-	// Unique ID associated to this credential. Should be a UUID.
-	ID string
 	credint.Container
 }
 
 type StoredCredential struct {
 	// This Key is generated by the storage module upon first write.
 	Key string `json:"key"`
 
-	// This is the same as `credential.id`. Typically, a URL that can be dereferenced.
-	CredentialID string `json:"credentialId"`
+	// ID of the credential that identifies it within ssi service.
+	ServiceID string `json:"serviceId"`
```

Comment:
perhaps something like `localCredentialID` and `externalCredentialID` may be more clear? or internal/external

Reply:
`externalCredentialID` is the same as `Credential.ID`. I don't think we should modify that because it's maps to the spec definition of `id`. 

Renamed this to `localCredentialID`.